.changeset/@envelop_newrelic-1935-dependencies.md

```diff
@@ -0,0 +1,8 @@
+---
+'@envelop/newrelic': patch
```

Comment:
@EmrysMyrddin this is really a patch? can we verify it doesn't require major?

Reply:
We don't drop the support for older versions, so I think patch is fine.

Reply:
Yea, it's why I updated manually instead of merging renovate PR :-)